client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Loggers.ts

```diff
@@ -17,7 +17,20 @@ export class ConsoleLogger implements ILogger {
 
     log(logLevel: LogLevel, message: string): void {
         if (logLevel >= this.minimumLogLevel) {
-            console.log(`${LogLevel[logLevel]}: ${message}`);
+            switch (logLevel) {
+                case LogLevel.Error:
+                    console.error(`${LogLevel[logLevel]}: ${message}`);
```

Comment:
@moozzyk, did we have any reason not to use console.error/warn/info calls instead of log?  Otherwise this looks good to me.

@RoBackman, thanks for the PR!

Reply:
The reason was that I only learned about console.error etc. after I merge the original PR and I did not want to add churn to alpha releases - so I opened #802